_data/changelog.yml

```diff
@@ -1,3 +1,8 @@
+- date: February 5, 2020
+  title: "TBD"
+  description: "TBD"
+  commit: TBD
```

Comment:
You'll need to refer to the specific git commit where you changed `terms-of-service.yml`. Note that we want 100% of changes to `terms-of-service.yml` in a single git commit so that we can link just one commit here. For that reason, you'll probably need to do a git rebase and merge your commits to create a single git commit. Once you have that,  you can link it here as part of the PR.